pkgs/test_api/pubspec.yaml

```diff
@@ -13,7 +13,6 @@ dependencies:
   collection: ^1.8.0
   meta: ^1.1.5
   path: ^1.2.0
-  pedantic: ^1.0.0
```

Comment:
This will need to be moved to `dev_dependencies` instead of dropped entirely. We still use it for analysis options.

Reply:
Done.  

I wasn't sure if I should bump any versions with this, so I'm leaving that for now but would be happy to add it and/or update a changelog if applicable.

Reply:
This looks fine, we'll drop the `-dev` and get everything ready to publish separately.